app/views/password_resets/new.html.haml

```diff
@@ -0,0 +1,15 @@
+- provide(:title, "Forgot password")
+
+%h1.forgot Forgot password
+
+.row
+  %img.logo-image-password-reset{:alt => 'Sunzi Bistro logo', :src => '/assets/logo.jpg'}/
+  
+  .col-md-6.col-md-offset-3
+    = form_with(url: password_resets_path, scope: :password_reset, local: true) do |f|
+
+      = f.label :email
+      = f.email_field :email, class: 'session form-control'
+
+      = f.submit "Submit", class: 'session btn btn-primary'
+      
```

Comment:
You should add a newline at the end of this file. Maybe your last line has an empty space and Github doesn't consider it an empty new line